presets/all-packages/src/attribute-group.ts

```diff
@@ -0,0 +1 @@
+export * from '@commercetools-test-data/attribute-group';
```

Comment:
This is how we re-export an internal package so it can be consumed independently.